helpers/utils.js

```diff
@@ -122,7 +122,7 @@ async function newContext(browser, options = {}) {
     }
 
     try {
-        const context = await browser.createIncognitoBrowserContext(options);
+        const context = await browser.createBrowserContext(options);
```

Comment:
why? this will share cookies between all contexts

Reply:
There is no `createIncognitoBrowserContext` method in puppeteer@22.15.0

Reply:
https://pptr.dev/api/puppeteer.browser.createbrowsercontext/#:~:text=This%20won%27t%20share%20cookies/cache%20with%20other%20browser%20contexts.